### DISCUSSION TITLE
point_0740

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#413] [Roy]
"20200608T084559_20200608T085154_T37UDB"
![image](https://user-images.githubusercontent.com/54947195/147897312-6b81e0d5-051b-4039-ae92-791b44c5b551.png)

"20190427T083601_20190427T083603_T37UDB"
![image](https://user-images.githubusercontent.com/54947195/147897398-64f558dd-cf08-490e-bfa1-ac86329ea958.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20190427T083601_20190427T083603_T37UDB"
![Captura](https://user-images.githubusercontent.com/86502862/154007179-ed2fea48-54d1-4d8e-a857-bca55fe2d336.JPG)
